config/board.ini

```diff
@@ -270,3 +270,43 @@ lib_ignore =
 extra_scripts =
     ${config:smallNoI2s.extra_scripts}
     pre:./scripts/get_git_rev.py
+
+
+; ********************************************************************************
+; M5Stack Core ESP32 WiFi and Bluetooth Module Development Board
+; ********************************************************************************
+[board:m5stack_core]
+extends = mcu:esp32, display:m5stack_core, config:normal
+board = m5stack-grey
+board_build.partitions = default_8MB.csv

```

Comment:
According to the pio board description it has 16 MB flash.
https://docs.platformio.org/en/stable/boards/espressif32/m5stack-grey.html

Reply:
Yes, forgot to change that - sorry